cli/lib/kontena/cli/nodes/list_command.rb

```diff
@@ -66,7 +66,7 @@ def node_data
         unless quiet?
           grid_health = grid_health(grid, grid_nodes)
           grid_nodes.each do |node|
-            node['name'] = health_icon(node_health(node, grid_health)) + " " + node['name']
+            node['name'] = health_icon(node_health(node, grid_health)) + " " + (node['name'] || '')
```

Comment:
I think it used to just get rendered as blank before the CLI table formatting changes, but yes, using something like that as a placeholder might be a good improvement.